.github/workflows/post-release.yml

```diff
@@ -0,0 +1,126 @@
+name: Post-Release
+
+permissions:
+  contents: read
+
+on:
+
+  workflow_run:
+    workflows: ["Release"]
+    types:
+      - completed
+  workflow_dispatch:
+    inputs:
+      alt_version:
+        required: false
+        description: Provide expected semver. If empty VERSION in repo will be used
+        default: ''
+
+env:
+  USE_VERSION: ${{ github.event.inputs.alt_version }}
+
+jobs:
+  installer_check:
+    name: Installer check
+    strategy:
+      matrix:
+        version: [none, stable, semver]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set the expected fluvio version
+        run: |
+          if [[ -z "${{ env.USE_VERSION }}" ]]; then
+            echo "EXPECTED_VERSION=$(cat VERSION)" | tee -a $GITHUB_ENV
+          else
+            echo "EXPECTED_VERSION=${{ github.event.inputs.alt_version }}" | tee -a $GITHUB_ENV
+          fi
+          echo "EXPECTED_VERSION: $EXPECTED_VERSION"
+
+      - name: Curl Install - no version
+        if: matrix.version == 'none'
+        run: curl -fsS https://packages.fluvio.io/v1/install.sh | bash | tee /tmp/installer.version
+      - name: Curl Install - stable 
+        if: matrix.version == 'stable'
+        run: curl -fsS https://packages.fluvio.io/v1/install.sh | VERSION=stable bash | tee /tmp/installer.version
+      - name: Curl Install - Version number
+        if: matrix.version == 'semver'
```

Comment:
all 3 steps seems to be same.  only diff is version which can be done in a separate step so action can be written as:
```
curl -fsS https://packages.fluvio.io/v1/install.sh | ${ env.FLUVIO_VERSION}  | tee /tmp/installer.version
```

where `FLUVIO_VERSIOn` can be computed in previous step

Reply:
I don't think I quite understand the goal you have in mind.

This matrix test is meant to validate the installer script installs the most recent release w/r/t the value of the `VERSION` env var, which is mainly how the installer script selects the version number to download. There are 3 cases I've identified.

The env var is:
* unset
  * The basic one-liner on the Fluvio site doesn't set this var
* set to `stable`
  * This is a release channel to test
* A specific version, e.g `x.y.z`
  * In our release process, this should be defined by the `VERSION` file. (But this is configurable in the action, for testing in between releases)

Reply:
My comment is about removing redundant expression: `curl -fsS https://packages.fluvio.io/v1/install.sh | xxx}  | tee /tmp/installer.version` for all 3 cases